versions/v0.12/antora-yml/antora-community.yml

```
@@ -0,0 +1,12 @@
+name: fleet
+title: Fleet
+version: next
+display_version: 'Next'
+start_page: index.adoc
```

```suggestion
start_page: en:index.adoc
```

Suggestion applies to all the newly introduced `antora-yml/*.yml` files. The value is currently invalid unless you specify the module (`en`) portion of the ID. You can only omit the module portion if using the ROOT module